src/ngraph_builder.cc

```diff
@@ -1810,6 +1810,18 @@ static Status TranslateDepthwiseConv2dNativeOp(
   return Status::OK();
 }
 
+static Status TranslateBroadcastDistributedOp(
```

Comment:
This should be guarded by `#if defined (NGRAPH_DISTRIBUTED)`

Reply:
fixed